OperationsUnitTest/OperationsUnitTest.cs

```diff
@@ -82,7 +82,7 @@ public void TestPerimeter()
         {
             //Arrange
             MathLibrary ClassOperation = new MathLibrary();
-            double expectedResult = 31.416;
+            double expectedResult = 31.41592653589793;
```

Comment:
you should check for a range: https://stackoverflow.com/questions/8825956/assert-areequal-with-system-double-getting-really-confusing
Because double are not always the same: https://www.jetbrains.com/help/resharper/CompareOfFloatsByEqualityOperator.html